Sources/PrinceOfVersions/ResponseModels/UpdateData/UpdateInfo.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  UpdateInfo.swift
+//  PrinceOfVersions
+//
+//  Created by Ivana Mršić on 29/05/2020.
+//
+
+import Foundation
+
+public protocol UpdateInfoValues {
+    var requiredVersion: Version? { get }
+    var lastVersionAvailable: Version? { get }
+    var installedVersion: Version { get }
+    var requirements: [String: Any]? { get }
+}
+
+public struct UpdateInfo {
+
+    /// Returns struct containing information about possible update.
+    public var updateData: UpdateInfoValues
+
+    /// Returns current SDK version.
+    public var sdkVersion: Version?
+
+    /**
+     Returns notification type.
+
+     Possible values are:
+     - Once: Show notification only once
+     - Always: Show notification every time app run
+
+     Default value is `.once`
```

Comment:
Change this into something like "If info is not provided by the configuration file, a default value will be set to `once`"